src/main/java/frc/robot/subsystems/Lift.java

```diff
@@ -434,14 +442,21 @@ public void periodic() {
       controlPosition(desiredPosition);
     }
 
-    // If the grabber is set to open and it is safe to open, open the grabber (drop). Otherwise,
-    // close it (grab).
-    if (!desiredGrabberClosed
-        && (armEncoder.getPosition() > Cal.Lift.GRABBER_CLOSED_ZONE_TOP_DEGREES
-            || armEncoder.getPosition() < Cal.Lift.GRABBER_CLOSED_ZONE_BOTTOM_DEGREES)) {
-      grabber.set(false); // drop
-    } else {
+    // // If the grabber is set to open and it is safe to open, open the grabber (drop). Otherwise,
+    // // close it (grab).
+    // if (!desiredGrabberClosed
+    //     && (armEncoder.getPosition() > Cal.Lift.GRABBER_CLOSED_ZONE_TOP_DEGREES
+    //         || armEncoder.getPosition() < Cal.Lift.GRABBER_CLOSED_ZONE_BOTTOM_DEGREES)) {
+    //   grabber.set(false); // drop
+    // } else {
+    //   grabber.set(true); // grab
+    // }
+    // TODO do we need a grabber zone?
```

Comment:
Do we have a reason to think it's no longer necessary?

Reply:
Changed grabber design